primitives/src/policy.rs

```diff
@@ -25,10 +25,10 @@ pub struct Policy {
 
 impl Policy {
     /// This is the address for the staking contract. Corresponds to
-    /// 'NQ38 STAK 1NG0 0000 0000 C0NT RACT 0000 0000'
+    /// 'NQ07 0000 0000 0000 0000 0000 0000 0000 0000'
```

Comment:
```suggestion
    /// 'NQ07 0000 0000 0000 0000 0000 0000 0000 0001'
```

Reply:
Yeah, I don't think having the zero address be the staking contract is a good idea.